lib/modules/userTagger.js

```diff
@@ -381,6 +381,11 @@ export class Tag {
 			// `defaultTagElement` is a lot fast than `buildTagElement`
 			instance.tagger = (this.text || this.color) ? Tag.buildTagElement(this) : Tag.defaultTagElement();
 			instance.tagger.addEventListener('click', () => this.openPrompt(instance));
+			// Remove the previously added tags
+			const previousTags = instance.element.getElementsByClassName(instance.tagger.className);
+			if (previousTags.length) {
+				previousTags.forEach(tag => tag.remove());
```

Comment:
Note that this misses the voteWeight element. It will also only remove previous tags added with `append`, not `after` (see beneath).

Since this is special behavior is only necessary for the (buggy?) postAuthor container, it might be cleaner if the workaround is moved up to the `watchForRedditEvents` listener.

Reply:
Okay. Thank you for pointing that out. I will make the change.